docs/how-to/testing-with-minio.md

```diff
@@ -0,0 +1,133 @@
+# Testing with Minio
```

Comment:
I think we should mention that this is about cos HA testing.

How about "Using Minio as S3 backend for Testing COS HA"?